Sentinel Lookback Mechanism (#44282)
demisto/content: Packs/AzureSentinel/Integrations/AzureSentinel/AzureSentinel_description.md

## Get the Additional Instance Parameters

To get the *Subscription ID*, *Workspace Name*, and *Resource Group* parameters, in the Azure Portal navigate to **Azure Sentinel** > your workspace > **Settings** and click the **Workspace Settings** tab.

## Lookback Parameter Notes

> * Increasing the **look-back** parameter value after incidents have already been fetched may cause duplicate incidents during the next fetch. To avoid duplicates, do not increase the look-back value once set.

Reviewer: Same as the comment on the readme

## Get the Additional Instance Parameters

> To get the *Subscription ID*, *Workspace Name*, and *Resource Group* parameters, in the Azure Portal navigate to **Azure Sentinel** > your workspace > **Settings** and click the **Workspace Settings** tab.

Reviewer 2: ```suggestion
1. In the Azure portal, navigate to **Azure Sentinel** > your workspace > **Settings**. and click the **Workspace Settings** tab.  
2. Click the **Workspace Settings** tab.  
3. Copy the **Subscription ID**, **Workspace Name**, and **Resource Group** parameters.  

```

> **look-back parameter note**:

Reviewer 2: ```suggestion
**Note**:
```

> In case the **look-back** parameter is initialized with a certain value and during a time that incidents were fetched, if changing

Reviewer 2: ```suggestion
Increasing the **look-back** parameter value after the initial fetch may result in duplicate incidents during the first run following the change. To avoid duplicates, ensure the lookback value is not increased once the integration is active.
```

> the look back to a number that is greater than the previous value, then in the initial incident fetching there will be incidents duplications.

Reviewer 2: ```suggestion
```

> If the integration was already set with look back > 0, and the look-back is not being increased at any point of time, then those incident duplications would not occur.

Reviewer 2: ```suggestion
```
